avaframe/com1DFAPy/DFAfunctionsCython.pyx

```diff
@@ -992,11 +989,11 @@ def computeGradC(cfg, particles, header, double[:, :] Nx, double[:, :] Ny,
         ux = 1
         uy = 0
         uz = -(1*nx + 0*ny) / nz
-        ux, uy, uz = normalize(ux, uy, uz)
+        nx, ny, nz = normalize(nx, ny, nz)
```

Comment:
should this not be `normalize(ux, uy, uz)`? 
Otherwise it is the same thing as line 985 and is not needed here...